src/components/ebay-breadcrumb/examples/2-lastpage-as-current/template.marko

```diff
@@ -0,0 +1,6 @@
+<ebay-breadcrumb aria-label="You are here">
```

Comment:
(minor) For `lastpage` examples, please separate it into `last-page` in the directory name so it reads correctly in the marketplace